nipype/interfaces/fsl/epi.py

```diff
@@ -354,14 +354,14 @@ class ApplyTOPUP(FSLCommand):
 
     >>> from nipype.interfaces.fsl import ApplyTOPUP
     >>> applytopup = ApplyTOPUP()
-    >>> applytopup.inputs.in_files = ["epi.nii", "epi_rev.nii"]
+    >>> applytopup.inputs.in_files = ["ds003_sub-01_mc.nii.gz", "ds003_sub-01_mc.nii.gz"]
```

Comment:
Oh I see. The ```_parse_inputs``` method is probably not the best place to this kind of check. I think it would be better placed the ```_run_interface``` method. Something like:

```
def _run_interface(self, runtime):
    ... your inputs sanity checks ...
    return super(ApplyTopup, self).__run_interface(runtime)
```
This way the doctests will pass with empty files and we don't alter the expected behavior of _parse_inputs.